<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GNSS Utility Library测试工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>数据输入区</string>
       </property>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>501</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>文件型测试数据</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>20</y>
           <width>481</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>数据文件路径：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtTestDataPath"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnTestDataPath">
            <property name="text">
             <string>…</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>501</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>简单数据输入区</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>481</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>请输入测试数据：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtSimpleTestData"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>140</y>
          <width>501</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>测试接口选择区</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>481</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>请选择测试接口：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboTestIF"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QPushButton" name="btnExecTest">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>210</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>执行测试</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>数据输出区</string>
       </property>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>501</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>文件型测试结果</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>481</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>结果输出文件夹路径：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtResultPath"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnResultPath">
            <property name="text">
             <string>…</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>501</width>
          <height>101</height>
         </rect>
        </property>
        <property name="title">
         <string>测试接口选择区</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>481</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>RTKLib执行结果：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtRtkResult"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>GULLib执行结果：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtGULResult"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
